client/client.go

```diff
@@ -246,8 +250,10 @@ func IsTwoFactorError(err error) bool {
 
 // SysInfo holds system information
 type SysInfo struct {
-	Series  string `json:"series,omitempty"`
-	Version string `json:"version,omitempty"`
+	Series    string `json:"series,omitempty"`
+	Version   string `json:"version,omitempty"`
+	OS        string `json:"os-id,omitempty"`
+	OnClassic bool   `json:"on-classic,omitempty"`
```

Comment:
That depends but it is what `/etc/os-release` will say until we choose to change it. For now I've dropped OnClassic (in SysInfo) entirely as it served little use apart from just being printed by `snap --version`